problemsets-mat223/linearalbegra.tex

```diff
@@ -1038,24 +1045,81 @@ \section*{Lines and Planes}
 	$z=y$.
 	\begin{parts}
 		\item Write $\ell$ in vector form. Is vector form of $\ell$ unique?
-		\item Write $L$ in vector form. 
+			\begin{solution}
+				$\vec x = t \mat{1 \\ -2} + \mat{0 \\ 3}$
+
+				The vector form is not unique, as any scalar multiple of the 
+				direction vector will work in place of $\mat{1 \\ -2}$, as will
+				any other point the line passes through work in place of 
+				$\mat{0 \\3}$. For example, another point on the line is 
+				$\mat{1 \\ 1}$, so the following vector equation describes the 
+				same line: $\vec x = t \mat{-4 \\ 8} + \mat{1 \\ 1}$.
+			\end{solution}
+		\item Write $L$ in vector form.
+			\begin{solution}[inline]
+				$\vec x = t\mat{1 \\ -2 \\ -2} + \mat{0 \\ 3 \\ 3}$
+			\end{solution}
 		\item Find another vector form for $L$ where both ``$\vec d$'' and
 			``$\vec p$'' are different from before.
+			\begin{solution}
+				Again, any multiple of the direction vector will work for 
+				$\vec d$, as will any other point on the line work for 
+				$\vec p$. For example, this equation describes the same 
+				line: $\vec x = t \mat{-3 \\ 6 \\ 6} + \mat{1 \\ 1 \\ 1}$.
+			\end{solution}
 	\end{parts}
 
 	\question
 	Let $A$, $B$, and $C$ be given in vector form by 
-	\[\overbrace{\vec x=t\mat{1\\2\\3}+\mat{0\\0\\1}}^{\displaystyle A}
+	\[
+	\overbrace{\vec x=t\mat{1\\2\\3}+\mat{0\\0\\1}}^{\displaystyle A}
 	\qquad \overbrace{\vec x=t\mat{-1\\1\\1}+\mat{-1\\1\\2}}^{\displaystyle B}
 	\qquad \overbrace{\vec x=t\mat{2\\-1\\1}+\mat{1\\1\\1}}^{\displaystyle C}.
 	\]
 	\begin{parts}
 		\item Do the lines $A$ and $B$ intersect? Justify your conclusion.
+			\begin{solution}
+				Yes. In this case this is quite easy to see since 
+				$\mat{0 \\ 0 \\ 1}$ is a point on line $A$, and this same point
+				 is on line $B$ (for the value $t = -1$).
+			\end{solution}
 		\item Do the lines $A$ and $C$ intersect? Justify your conclusion.
+			\begin{solution}
+				No. In order for them to intersect, there would have to exist 
+				two real numbers $a$ and $c$ such that when substituded for $t$ 
+				in the equations of $A$ and $C$, respectively, the resulting 
+				vectors would equal. In other words, we would have to have:
+				\[
+					a\mat{1\\2\\3}+\mat{0\\0\\1} = c\mat{2\\-1\\1}+\mat{1\\1\\1}.
+				\]
```

Comment:
We can make this more consise. "No. The vector equation ... has no solution since.... Therefore the lines cannot intersect"